ers/sql.go

```diff
@@ -169,13 +342,14 @@ func (rdr *SQLEventReader) readLoop(db *gorm.DB, sqlDB io.Closer) {
 			for i := range columns {
 				columnPointers[i] = &columns[i]
 			}
+			// copy row values to their respective column
 			if err = rows.Scan(columnPointers...); err != nil {
 				rdr.rdrErr <- err
 				rows.Close()
 				return
 			}
 			msg := make(map[string]any)
-			fltr := make(map[string]string)
+			fltr := make(map[string]any)
```

Comment:
If possible, please redefine the filter related variables with more suggestive names.